src/modules/discourse/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { Discourse, DiscoursePostInTopic } from '../../clients/Discourse'
+
+export async function fetchAllComments(discourseTopicId: number) {
```

Comment:
I think this can be part of DiscourseService 🤔

Reply:
I tried, but the service is not avail to the front :/